schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3299 @@
+{
+  "name": "ClinVar",
+  "schema": {
+    "tables": [
+      {
+        "name": "clinvar_variation_release",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "clinvar_variation_release",
```

Comment:
Would it be a huge pain to change the pattern we use for ID columns? i.e. every time we have a column of IDs pointing to another table, we rename the column to `<table-name>_id` or `<table-name>_ids`. Not sure how easy that will be to do mechanically at ETL time...